.github/workflows/quiz-manager.yml

```diff
@@ -0,0 +1,25 @@
+
+name: quiz-manager
+on: [pull_request]
+jobs:
+  my_review:
+    name: quiz-manager review
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Checkout systems-cs-pub-ro/quiz-manager
+      uses: actions/checkout@v3
+      with:
+        repository: systems-cs-pub-ro/quiz-manager
+        token: ${{ secrets.GITHUB_TOKEN }}
+        path: quiz-manager
+    - id: files
+      uses: jitterbit/get-changed-files@v1
+      continue-on-error: true
+      with:
+        format: 'csv' 
+    - run: |
+        mapfile -d ',' -t added_modified_files < <(printf '%s,' '${{ steps.files.outputs.added_modified }}')
+        for added_modified_file in "${added_modified_files[@]}"; do
+          python3 quiz-manager/src/quiz_manager.py check -t common -i ${added_modified_file}
+        done
```

Comment:
Will this job fail if one or more checked files don't have the appropriate syntax?

I think you should exit with code 1 if at least one file fails.

https://docs.github.com/en/actions/creating-actions/setting-exit-codes-for-actions

Reply:
The job fails regardless. When at least one file has the wrong format, the action is stopped and an error occurs:

```console
Run mapfile -d ',' -t added_modified_files < <(printf '%s,' 'my-hr.hr,my-hr2.hr')
my-hr.hr, line 1: Wrong format in metadata line!

Error: Process completed with exit code 1.
```

This is an example I've done on one of my repos, with multiple files (`my-hr.hr` and `my-hr2.hr`) were`my-hr.hr` is having bad syntax.

Reply:
What happens if both files have syntax problems? Will it only show the problems in the first encountered file?

Reply:
Yes, unfortunately. Not ideal at all.

Reply:
I suggest finding a way to show all errors in one pass. If you don't find a solution for this, you can still merge this.